server/android/src/android.rs

```diff
@@ -107,6 +109,8 @@ pub mod android {
         if !cert_path.join(PRIVATE_CERT_FILE).exists() {
             generate_certs(&cert_path);
         }
+        let hardware_id: String = env.get_string(android_id).unwrap().into();
+        info!("Android starts.. {}", hardware_id);
```

Comment:
left in simply to avoid getting a warning about not using my variables. I expect that this will be changed soon.

Reply:
Just in case you didn't know, you can avoid a warning for an unused variable, by starting the variable with '_' e.g. `let _hardward_id = env.get_string...`

Not sure why you need to .into() here, to be more consistent with our other code you might want something like
`let hardware_id = env.get_string(android_id).unwrap().to_string();`
(If that actually works, I haven't tried it.)

*I think* we're been preferring to_string() over other options as it makes it readable. Other options are things like `String::from(<CODE>)` and you can use `.to_owned()` where `.to_string()` would work but that isn't as clear to people new to rust (IMAO)

Reply:
thanks! and yep, I do use `_` generally, but in this case just rolled with what was already there. Essentially, I was over it after spending so long on the changes and since it was working.. didn't want to touch it again in case I broke it! 🤣 like I say, I expect the hardware_id to be reworked in the near future.  But have popped it in anyway and removed the info call 🤷 

The `into` was just following the existing pattern here - I prefer `to_string` for readability - it says what it does, to me anyway

Reply:
For what it's worth, logging things like this are a good idea IMAO, like we discussed yesterday, having more logs is going to be helpful in future debugging efforts.

Single log entries on startup that provide useful information are generally good, if it was logging on every request, then it would be very annoying.